tests/IBFE/explicit_ex5.cpp

```diff
@@ -732,7 +732,7 @@ postprocess_data(Pointer<Database> input_db,
         // This computed value is not stable between debug and optimized mode
         // (and also different versions of libMesh): make it slightly more
         // stable by increasing the order of magnitude
-        pout << loop_time << " " << 0.01 + -F_integral[1] / (0.5 * rho * U_max * U_max * D) << endl;
+        pout << loop_time << " " << 0.01 + std::abs(F_integral[1] / (0.5 * rho * U_max * U_max * D)) << endl;
```

Comment:
Why is this the right thing to do?

Reply:
In general? I don't remember the details but changing the version of libMesh gave us results different enough to upset numdiff.

In this case? The value goes negative in the newish ex5 regridding test. Since the value passes near zero this old trick of adding a value to it isn't enough to get consistent output.

Reply:
You could just look at `F_integral[0]` --- that should be nonnegative.

Reply:
That works.